src/main/resources/idprovider/idprovider.js

```diff
@@ -15,7 +27,19 @@ exports.handle401 = function() {
     };
 };
 
-exports.get = function() {
+exports.get = function (req) {
+    if (!req.rawPath.startsWith('/api/idprovider/')) {
+        const indexOf = req.rawPath.indexOf('/_/');
+        if (indexOf !== -1) {
+            const endpointPath = req.rawPath.substring(indexOf)
+            if (/^\/_\/idprovider\/[^/]+\/_static\/.+$/.test(endpointPath)) {
+                return getStatic(req);
+            }
+        }
+    } else if (/^\/api\/idprovider\/[^/]+\/_static\/.+$/.test(req.rawPath)) {
```

Comment:
We should make it a bit more clear what all these conditions in "if" mean. I suggest we do something like this:
```

const isStaticAssetLocal = /^\/_\/idprovider\/[^/]+\/_static\/.+$/.test(endpointPath);
const isStaticAssetApi = /^\/api\/idprovider\/[^/]+\/_static\/.+$/.test(req.rawPath);

...

if (!isStaticAssetApi) {
...
} else if (isStaticAssetLocal) {
...
}
```

Reply:
We must cut `baseUri` for non API case, because for us interesting only part after `/_/`